Makefile

```diff
@@ -0,0 +1,38 @@
+YARPC=yarpc
+TEST_DIR=tests/unit
+
+.PHONY: create-env
+create-env:
+	rm -rf env
+	python3 -m pip install --user virtualenv
+	python3 -m venv env
```

Comment:
`python3`, `pip` and `pytest` should be moved to top as global vars: `PYTHON ?= python3`